content/blog/2019-06-04-rromeo.md

```diff
@@ -0,0 +1,169 @@
+But several very different models exist under this umbrella term of "Open Access".
+One of them is the so-called ["green open access"](https://en.wikipedia.org/wiki/Open_access#Green_OA), where the articles are made publicly available by their authors via their deposition in institutional or public repositories, such as [bioRxiv](https://www.biorxiv.org) or [HAL](https://hal.archives-ouvertes.fr/).
+Scientific journals have different policies regarding green open access: some let you archive various versions of the manuscript right after acceptance, while others ask you to wait an embargo period or forbid entirely the archival of the manuscript.
+To support green open access, scientists can elect to publish their work in journals that authorize manuscript archival.
+And this is where [SHERPA/RoMEO](http://sherpa.ac.uk/romeo/index.php) comes in handy: it offers a publicly available database of open access policies of scientific journals and lists the conditions under which manuscript archival might be allowed.
+
+The SHERPA/RoMEO database is available through an [Application Programming Interface](https://en.wikipedia.org/wiki/Application_Programming_Interface) (API), which meant that we could build an R client to programmatically access this data.
+This would allow researchers to more easily select journals based on their manuscript archival policies. An R client would also be a precious tool for bibliometricians who want to get the general picture of open access practices in a particular subfield.
+
+Although we had no prior experience working with web data or scraping APIs for data, we knew several tools existed to interact with APIs in R such as [httr](https://github.com/r-lib/httr) or rOpenSci's [crul](https://github.com/ropensci/crul).
+And as we started the development of [rromeo](https://github.com/ropensci/rromeo), we also knew we could use the many examples of R client for APIs available on the [CRAN Task View on Web Technologies](https://cran.r-project.org/web/views/WebTechnologies.html) as models.
+
+### SHERPA/RoMEO API
+
+SHERPA/RoMEO has been available through a web interface since at least February 2004 according to the [Internet Archive](https://archive.org/) and their API was released in December 2006 (from the Internet Archive <https://web.archive.org/web/20061211125806/http://www.sherpa.ac.uk:80/romeo.php>), making their data available to anyone since then.
+
+There are three versions of the manuscript considered in SHERPA/RoMEO:
+
+1. pre-print, which is the manuscript version **before peer review**;
+1. post-print, which is the manuscript as **accepted after peer review** but not yet typeset as an article in the journal;
+1. and typeset manuscript/publisher's manuscript, which is the manuscript with the text after peer-review, **fully typeset**, as it appears in the journal.
+
```

Comment:
Hi @stefaniebutland :wave:, we would like to add an image here to explain more clearly the difference between pre-prints, post-prints and publisher's pdf. We have identified two good visuals but we can't choose between them. An external opinion would be great!

https://paleorxiv.github.io/article_types.jpg (CC BY)

or

https://s3.amazonaws.com/libapps/accounts/3099/images/Publishingprocess2.PNG (public domain)

What do you think?

Reply:
I like the clarity of the first one